frontend/src/store/authStore.ts

```diff
@@ -13,6 +13,7 @@ export const useAuthStore = defineStore('auth', () => {
   const ready = ref(false);
 
   // Getters
+  const getKeycloak = computed(() => $keycloak.value);
   const getAuthenticated = computed((): boolean => (ready.value ? !!$keycloak.value.authenticated : false));
   const getLoginUrl = computed(() => (ready.value ? $keycloak.value.createLoginUrl() : ''));
   const getToken = computed(() => (ready.value ? $keycloak.value.token : ''));
```

Comment:
If everything is going through getKeycloak now, maybe we can remove these other 3 getters?